specific-versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Enter script located dir, whaterver the script is called from
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+if [ -z ${TF_VAR_secret_key+x} ]; then echo "TF_VAR_secret_key is unset"; exit 1; else echo "TF_VAR_secret_key is set to '$TF_VAR_secret_key'"; fi
+if [ -z ${TF_VAR_access_key+x} ]; then echo "TF_VAR_access_key is unset"; exit 1; else echo "TF_VAR_access_key is set to '$TF_VAR_access_key'"; fi
```

Comment:
@Sammers21 this check is duplicated without good reason

Reply:
@olegmoz, no duplications `TF_VAR_secret_key `and `TF_VAR_access_key` are different variables

Reply:
@Sammers21 oh, sorry, my bad